integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.html

```diff
@@ -0,0 +1,31 @@
+<td class="center" *ngIf="!isFlagquote">
+  <ng-container
+    *ngIf="
+      quoteDiscountData &&
+      quoteDiscountData.basePrice?.value !== undefined &&
+      quoteDiscountData.cpqDiscounts &&
+      quoteDiscountData.cpqDiscounts.length > 0
+    "
+  >
+    <ng-container *ngFor="let discount of quoteDiscountData.cpqDiscounts">
+      <span role="cell" class="cx-total cx-discount">
+        {{ discount.isoCode
+        }}{{
+          getDiscountedPrice(quoteDiscountData.basePrice?.value, discount.value)
+        }}
+      </span>
+    </ng-container>
+  </ng-container>
+  <!-- Iterate over quoteDiscountData.basePrice -->
+  <span
+    class="cx-formattedValue"
```

Comment:
Please, do not use camel case for the class. Rename the class, for instance to `cx-formatted-value`